src/os/bluestore/KernelDevice.cc

```diff
@@ -46,6 +46,7 @@ KernelDevice::KernelDevice(aio_callback_t cb, void *cbpriv)
 {
   zeros = buffer::create_page_aligned(1048576);
   zeros.zero();
+  rotational = true;
```

Comment:
no problem :). I will remember at least this one get merged properly with dependent changes,